packages/block-editor/src/components/block-list/block.js

```diff
@@ -592,12 +594,27 @@ function BlockListBlockProvider( props ) {
 				hasBlockSupport: _hasBlockSupport,
 				getActiveBlockVariation,
 			} = select( blocksStore );
+
 			const attributes = getBlockAttributes( clientId );
 			const { name: blockName, isValid } = blockWithoutAttributes;
 			const blockType = getBlockType( blockName );
 			const { supportsLayout, __unstableIsPreviewMode: isPreviewMode } =
 				getSettings();
 			const hasLightBlockWrapper = blockType?.apiVersion > 1;
+			const defaultClassNames = [];
+			if ( hasLightBlockWrapper && blockName ) {
```

Comment:
Is there a case where `blockName` isn't set?

Reply:
Hmm, good point. Seems unlikely. Probably another leftover from a previous iteration of the PR.

I've removed it in 8ea9ac35be31089fd97546012bfb9368c5d3f200.